bundles/org.eclipse.chronograph.api/META-INF/MANIFEST.MF

```diff
@@ -7,4 +7,8 @@ Bundle-Name: %Bundle-Name
 Bundle-Vendor: %Bundle-Vendor
 Bundle-Copyright: %Bundle-Copyright
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Export-Package: org.eclipse.chronograph.api
+Export-Package: org.eclipse.chronograph.internal.api,
+ org.eclipse.chronograph.internal.api.adapters,
+ org.eclipse.chronograph.internal.api.builders,
+ org.eclipse.chronograph.internal.api.providers
+Import-Package: org.eclipse.swt.graphics
```

Comment:
Let's avoid this - we should not depend on particular graphical library

Reply:
No objection, dependency was removed.